tests/simulator/per_test.h

```diff
@@ -83,25 +83,33 @@ struct TestSimulator : public HierarchyMaker<_dim>,
 };
 
 template<typename Simulator>
-struct SimulatorTest : public ::testing::Test
+struct /*[[deprecated]]*/ SimulatorTest : public ::testing::Test
```

Comment:
do we keep that one?

Reply:
it's still in use so in this PR, I think yes.